xt/41-coaload.t

```diff
@@ -1,6 +1,7 @@
 use Test::More;
 use strict;
 use warnings;
+use LedgerSMB::Sysconfig;
```

Comment:
Ok. I'll add PGHOST and PGPORT in that section and, after merging #2635, close this one.